### PR TITLE
Update templates tags to improve the template picker

### PR DESCRIPTION
**Pull request by lolautruche**

### CODE REVIEW

File: templates/django4/.platform.template.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ info:
   tags:
   - Python
   - Framework
+  - Django
```
[the remark this answers is not in the document]
Comment (lolautruche) [reply templates/django4/.platform.template.yaml:32]: Yes, similar to Symfony, or other PHP frameworks. This is for the template picker to get the appropriate filters.

File: templates/typo3/.platform.template.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ info:
   tags:
   - PHP
   - CMS
+  - Typo3
```
Comment (flovntp) [inline templates/typo3/.platform.template.yaml:12]: As for Sylius, Typo3 is using Symfony (components only) so i don't know if we need to make SF appears here... 
if yes, it should appears also for Drupal templates, right ?

Comment (lolautruche) [reply templates/typo3/.platform.template.yaml:12]: Ah I wasn't aware of this. Yes, Symfony might appear here as well, but I'm not sure this makes sense (same for Drupal)

Comment (lolautruche) [reply templates/typo3/.platform.template.yaml:12]: Sylius is Symfony full stack, Drupal and Typo3 are not

Comment (flovntp) [reply templates/typo3/.platform.template.yaml:12]: true, so lets forget about all those comments...

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**